Fix tests/integration_tests/cli/test_integration_cli.py::test_failing_job_cli_error_message

**Describe the bug**
The following test fails, but only when running it as github workflow:
```python
def test_failing_job_cli_error_message():
        # modify poly_eval.py
        with open("poly_eval.py", mode="a", encoding="utf-8") as poly_script:
            poly_script.writelines(["    raise RuntimeError('Argh')"])
    
        args = Mock()
        args.config = "poly_high_min_reals.ert"
        parser = ArgumentParser(prog="test_main")
    
        parser = ArgumentParser(prog="test_main")
        parsed = ert_parser(
            parser,
            [TEST_RUN_MODE, "poly.ert"],
        )
        expected_substrings = [
            "Realization: 0 failed after reaching max submit (2)",
            "job poly_eval failed",
            "Process exited with status code 1",
            "Traceback",
            "raise RuntimeError('Argh')",
            "RuntimeError: Argh",
        ]
        try:
            run_cli(parsed)
        except ErtCliError as error:
            for substring in expected_substrings:
>               assert substring in f"{error}"
E               AssertionError: assert 'Realization: 0 failed after reaching max submit (2)' in 'Experiment failed!\n'
```

**To reproduce**
It only fails on github action workflows.
more info here for example: https://github.com/equinor/komodo-releases/actions/runs/7352338393/job/20017761397

**Expected behaviour**
The test should pass

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment**
 - OS: [e.g. RHEL7]
 - ERT/Komodo release: bleeiding
 - Python version
 - Remote/HPC execution involved: no

**Additional context**
This test fails when running with job_queue while scheduler execution  is skipped. Apparently the error message is not composed correctly.

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -1,5 +1,6 @@
 import asyncio
 import fileinput
+import logging
 import os
 import pkgutil
 import resource
@@ -28,6 +29,18 @@ from .utils import SOURCE_DIR
 
 if TYPE_CHECKING:
     from importlib.abc import FileLoader
+
+
+@pytest.fixture(autouse=True)
+def log_check():
+    logger = logging.getLogger()
+    logger.setLevel(logging.WARNING)
+    yield
+    logger_after = logging.getLogger()
+    level_after = logger_after.getEffectiveLevel()
+    assert (
+        level_after == logging.WARNING
+    ), f"Detected differences in log environment: Changed to {level_after}"
 
 
 @pytest.fixture

--- a/tests/unit_tests/conftest.py
+++ b/tests/unit_tests/conftest.py
@@ -1,22 +1,9 @@
-import logging
 import os
 import sys
 
 import pytest
 
 from ert.ensemble_evaluator.config import EvaluatorServerConfig
-
-
-@pytest.fixture(autouse=True)
-def log_check():
-    logger = logging.getLogger()
-    logger.setLevel(logging.WARNING)
-    yield
-    logger_after = logging.getLogger()
-    level_after = logger_after.getEffectiveLevel()
-    assert (
-        level_after == logging.WARNING
-    ), f"Detected differences in log environment: Changed to {level_after}"
 
 
 @pytest.fixture(autouse=True)

